wallet/src/ui/app/redux/slices/app/index.ts

```diff
@@ -18,12 +18,16 @@ type AppState = {
     appType: AppType;
     apiEnv: API_ENV;
     navVisible: boolean;
+    activeOrigin: string | null;
```

Comment:
This is fine for now, but I wonder if should be some kind of state bag for the sake of easily adding other information beyond the origin and favicon.